README.md

```diff
@@ -118,6 +118,42 @@ class ContentApp extends Component {
 }
 ```
 
+### 4. Optional: Retrieve information about the initiator of the action
+
+There are probably going to be times where you are going to want to know who sent you a message. For example, maybe you have a content script and you want to have it send information to a store that is managed by the background script and you want your background script to know which tab sent the information to it. You can retrieve this information by using the `_sender` property of the action. Let's look at an example of what this would look like.
```

Comment:
I believe (IIRC) that @tshaddix prefers using UI Component in place of "content script" wherever possible. (correct me if I'm wrong Tyler)

This looks dope though!